node/tests/RedisClusterClient.test.ts

```diff
@@ -297,4 +298,41 @@ describe("RedisClusterClient", () => {
         },
         TIMEOUT,
     );
+
+    it.each([ProtocolVersion.RESP2, ProtocolVersion.RESP3])(
+        `check that multi key command returns a cross slot error`,
+        async (protocol) => {
+            const client = await RedisClusterClient.createClient(
+                getOptions(cluster.getAddresses(), protocol),
+            );
+
+            await checkCommandThrowsCrossSlotError(
+                client.brpop(["abc", "zxy", "lkn"], 0.1),
+            );
+            await checkCommandThrowsCrossSlotError(
+                client.blpop(["abc", "zxy", "lkn"], 0.1),
+            );
+            await checkCommandThrowsCrossSlotError(client.rename("abc", "zxy"));
```

Comment:
@shohamazon how can I do a loop here like I did in python client tests?

Reply:
Can you reference me to an example of that in python?

Reply:
See changes in `test_async_client.py` in this PR
https://github.com/aws/glide-for-redis/pull/1391/files#diff-1c3a6923c1bf3dd19a1c98c6523f881876a156a8316c3b3f5dc6801a405382ceR2535-R2544

Reply:
ohh sorry I was on the app through my phone so I didnt see the full pr, taking a look

Reply:
possible:

```
const promises = [
                    client.blpop(["abc", "zxy", "lkn"], 0.1),
                    client.rename("abc", "zxy"),
                    client.brpop(["abc", "zxy", "lkn"], 0.1),
                    client.rename("abc", "zxy"),
                    client.zdiffstore("abc", ["zxy", "lkn"]),
                    client.zrangestore("abc", "zxy", [0, -1]),
                ];

                for (const promise of promises) {
                    try {
                        await promise;
                    } catch (e) {
                        expect((e as Error).message).toContain("Crrosslot");
                    }
                }
```

Reply:
This worked! Much appreciated!